desktop_native/src/password/windows.rs

```diff
@@ -140,19 +140,14 @@ fn target_name(service: &str, account: &str) -> String {
 mod tests {
     use super::*;
 
-    #[tokio::test]
-    async fn test() {
-        set_password("BitwardenTest", "BitwardenTest", "Random")
-            .await
-            .unwrap();
+    #[test]
+    fn test() {
```

Comment:
Can we get a test of `get_password_keytar` for the windows version?

Reply:
That means we would need to implement the store method as well 😢

Reply:
Actually I think it would be better to have a test dep on [keytar](https://crates.io/crates/keytar) that stores it for us and we can prove we read it properly.

That way, if Keytar ever updates, we'll fail